Code/GraphMol/testChirality.cpp

```diff
@@ -2818,18 +2818,18 @@ class TestAssignChiralTypesFromMolParity {
         {Atom::CHI_UNSPECIFIED, 0},
         {Atom::CHI_OTHER, 0}};
     MolOps::assignChiralTypesFrom3D(*d_rwMol);
-    for (auto ai = d_rwMol->beginAtoms(); ai != d_rwMol->endAtoms(); ++ai) {
-      int parity = parityMap.at((*ai)->getChiralTag());
-      (*ai)->setProp(common_properties::molParity, parity);
-      (*ai)->setChiralTag(Atom::CHI_UNSPECIFIED);
+    for (const auto atom : d_rwMol->atoms()) {
+      int parity = parityMap.at((atom)->getChiralTag());
```

Comment:
you don't need the parens around `atom` anymore, so you can do:
```suggestion
      int parity = parityMap.at(atom->getChiralTag());
```
here and below